brave/src/main/java/brave/handler/FinishedSpanHandler.java

```diff
@@ -64,9 +65,60 @@
    * visible to later handlers, including Zipkin.
    * @return true retains the span, and should almost always be used. false drops the span, making
    * it invisible to later handlers such as Zipkin.
+   *
+   * @see #supportsOrphans() If you are scrubbing personal information, consider supporting orphans.
    */
   public abstract boolean handle(TraceContext context, MutableSpan span);
 
+  /**
+   * Normally, {@link #handle(TraceContext, MutableSpan)} is only called upon explicit termination
+   * of a span: {@link Span#finish()}, {@link Span#finish(long)} or {@link Span#flush()}. When this
+   * method returns true, the callback will also receive data orphaned due to spans being never
+   * terminated or data added after termination. It is important to understand this, especially if
+   * your handler is performing work like redaction. This sort of work needs to happen on all data,
+   * not just the success paths.
+   *
+   * <p><h3>What is an orphaned span?</h3>
```

Comment:
If this is zipkin style it's fine to be consistent, but would normally expect to not put the `h3` in the `p` (think browsers move the h3 out of p tags)

```java
<h3>Hello?</h3>

<p>World
```

Reply:
this was just me fighting intellij's quick documentation thing recently. not the best thing to optimize for. you are definitely right!